### ISSUE TITLE
What's in a name?

### ISSUE THREAD

**Issue by studiomohawk** [frontendweekly/feeds#4720]
[What's in a name?](https://sarahmhigley.com/writing/whats-in-a-name/) / Often when we talk about accessibility problems, we end up talking about a number of different errors that all boil down to a missing accessible name: form fields without labels, images without alts, icon buttons without readable text, and many more.
